main.nf

```diff
@@ -1276,14 +1318,16 @@ process VARSCAN2 {
 
     output:
     tuple val(sample), val(single_end), path("${prefix}.vcf.gz*") into ch_varscan2_highfreq_consensus,
-                                                                       ch_varscan2_highfreq_snpeff
+                                                                       ch_varscan2_highfreq_snpeff,
+                                                                       ch_varscan2_highfreq_intersect
     tuple val(sample), val(single_end), path("${sample}.vcf.gz*") into ch_varscan2_lowfreq_snpeff
     path "${prefix}.bcftools_stats.txt" into ch_varscan2_bcftools_highfreq_mqc
     path "*.varscan2.log" into ch_varscan2_log_mqc
     path "${sample}.bcftools_stats.txt"
 
     script:
     prefix = "${sample}.AF${params.max_allele_freq}"
+    strand = params.protocol != 'amplicon' && params.varscan2_strand_filter ? "--strand-filter 1" : "--strand-filter 0"
```

Comment:
I typically add `def` I did not know actually that it works without
```suggestion
    def strand = ( params.protocol != 'amplicon' && params.varscan2_strand_filter ) ? "--strand-filter 1" : "--strand-filter 0"
```

Reply:
I think the jury is still out on when to use `def` appropriately 🙂 In a variable context, I tend to use it when I have to initialise a variable that has to be overwritten e.g. [here](https://github.com/nf-core/viralrecon/blob/7c64c35157267e8977bdd7b72a6227c2c84843a5/main.nf#L358).